Source/Particles/MultiParticleContainer.cpp

```diff
@@ -756,14 +756,24 @@ void MultiParticleContainer::InitQuantumSync ()
 
     //If specified, use a user-defined energy threshold for photon creaction
     ParticleReal temp;
+    constexpr auto mec2 = PhysConst::c * PhysConst::c * PhysConst::m_e;
     if(pp.query("photon_creation_energy_threshold", temp)){
+        temp *= mec2;
         m_quantum_sync_photon_creation_energy_threshold = temp;
     }
     else{
         amrex::Print() << "Using default value (2*me*c^2)" <<
             " for photon energy creaction threshold \n" ;
     }
 
+    // qs_minimum_chi_part is the minium chi parameter to be
+    // considered for Synchrotron emission. If a lepton has chi < chi_min,
+    // the optical depth is not evolved and photon generation is ignored
+    amrex::Real qs_minimum_chi_part;
+    if(!pp.query("chi_min", qs_minimum_chi_part))
+        amrex::Abort("qed_qs.chi_min should be provided!");
```

Comment:
```suggestion
    pp.get("chi_min", qs_minimum_chi_part);
```

You can use get for this. Although it is true that the error message when get fails is not as clear and visible.